src/main/resources/merge_orchestrator.sql

```diff
@@ -0,0 +1,51 @@
+
+    create table configs (
+       processUuid uuid not null,
+        businessProcess varchar(255),
+        process varchar(255),
+        runBalancesAdjustment boolean,
+        primary key (processUuid)
+    );
+
+    create table IgmEntity_boundaries (
+       IgmEntity_date timestamp not null,
+        IgmEntity_processUuid uuid not null,
+        IgmEntity_tso varchar(255) not null,
+        boundary uuid
```

Comment:
primary key?

Reply:
Tables 'IgmEntity_boundaries' and 'ProcessConfigEntity_tsos' are automatically created from IgmEntity and ProcessConfigEntity.
Things are similar in actions-server with the countries in the FilterContingencyList